test/e2e/cluster_upgrade.go

```diff
@@ -196,7 +196,9 @@ var _ = Describe("Skipped", func() {
 		svcName, replicas := "baz", 2
 		var rcName, ip, v string
 		var ingress api.LoadBalancerIngress
-		f := Framework{BaseName: "cluster-upgrade"}
```

Comment:
But thank you for bringing this inappropriate use of framework to my attention... these should be changed to use the constructor.